### PR TITLE
Add npm to our runners bundle

### PR DESCRIPTION
**Pull request by valentyn88**
Add npm to our runners bundle

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -63,3 +63,6 @@ jobs:
       - 
         name: List bundle contents
         run: ls -R bundle/
+      -
+        name: Print npm version
+        run: npm -v
```
Comment (dpgraham) [inline .github/workflows/test.yml:68]: This version of `npm` would be the version that's on the path and included with the Windows GitHub Actions images.

I think the `npm` package that you added to `package.json` would be located at `./node_modules/?` (maybe `./node_modules/.bin/npm`?)

Comment (valentyn88) [reply .github/workflows/test.yml]: Thanks! Here is the proper path bundle/node_modules/.bin/npm -v

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "js-yaml": "3.14.0",
     "lodash": "4.17.20",
     "mocha-junit-reporter": "^2.0.0",
+    "npm": "^6.14.9",
```
Comment (alexplischke) [inline package.json:25]: I'd recommend pinning it down (that also goes for all the other dependencies, but you don't have to change those as part of your PR):

```suggestion
    "npm": "6.14.9",
```

Comment (valentyn88) [reply package.json]: Good idea! I did the same for other dependencies as well.

**Review by dpgraham — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

**Comment by alexplischke** [comment]
@dpgraham Is this enough to get NPM working on VMs? Or is there more to be done on the image side of things?

**Comment by dpgraham** [comment]
@alexplischke this should be sufficient, yes. The test in GitHub Actions should sufficiently prove that it works. I suppose another test that can be added, on top of just printing the version, would be to have it install a package, something like `npm install lodash`

**Pull request merged**